<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VesselPanel</class>
 <widget class="QWidget" name="VesselPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>84</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border:0px transparent solid;background-color:rgb(46,47,48);color:white;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>41</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <item>
      <property name="text">
       <string>删除</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>跳转</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="vesselSize" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
      <item>
       <widget class="QPushButton" name="largeVessel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>大血管直径100um</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{background-color:rgb(0, 187, 163);}</string>
        </property>
        <property name="text">
         <string>大</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="middleVessel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>中血管直径50um</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{background-color:rgb(0, 187, 163);}</string>
        </property>
        <property name="text">
         <string>中</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="smallVessel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>小血管直径20um</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{background-color:rgb(0, 187, 163);}</string>
        </property>
        <property name="text">
         <string>小</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="vessleEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="placeholderText">
      <string>输入直径，按回车确认</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
